boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_cpunet_reset.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <init.h>
+#include <logging/log.h>
+
+#include <soc.h>
+#include <drivers/gpio.h>
+
+#include "nrfx_gpiote.h"
+
+LOG_MODULE_REGISTER(nrf5340_audio_dk_nrf5340_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
+
+#if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE)
+
+#if defined(CONFIG_BT_CTLR_DEBUG_PINS_CPUAPP)
+#include <../subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h>
+#else
+#define DEBUG_SETUP()
+#endif
+
+#define SHARED_SPI DT_NODELABEL(spi4) /* SD card and HW codec share the SPI4 */
+
+static int core_config(void)
+{
+	int ret;
+	const struct device *gpio_dev;
+	nrf_gpiote_latency_t latency;
+
+	latency = nrfx_gpiote_latency_get();
+
+	if (latency != NRF_GPIOTE_LATENCY_LOWPOWER) {
+		LOG_DBG("Setting gpiote latency to low power");
+		nrfx_gpiote_latency_set(NRF_GPIOTE_LATENCY_LOWPOWER);
+	}
+
+	/* Workaround for issue with PCA10121 v0.7.0 related to SD-card */
+	gpio_dev = device_get_binding(DT_SPI_DEV_CS_GPIOS_LABEL(DT_NODELABEL(sdhc0)));
+	if (!gpio_dev) {
+		return -ENODEV;
```

Comment:
please use `DEVICE_DT_GET`

Reply:
Removed entire gpio_dev from this file, due to using GPIO_DT_SPEC_GET and nrf_gpio_reconfigure